src/tests/work-item/work-item-list/assigne.spec.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * POC test for automated UI tests for ALMighty
+ *  Story: Display and Update Work Item Details
+ *  https://github.com/almighty/almighty-core/issues/298
+ *
+ * Note on screen resolutions - See: http://www.itunesextractor.com/iphone-ipad-resolution.html
+ * Tests will be run on these resolutions:
+ * - iPhone6s - 375x667
+ * - iPad air - 768x1024
+ * - Desktop -  1920x1080
+ *
+ * beforeEach will set the mode to phone. Any tests requiring a different resolution will must set explicitly.
+ *
+ * @author ldimaggi
```

Comment:
I am honored to be included in the source code for this test, but the comments should list you as the author.   ;-)